vllm/v1/worker/gpu/async_utils.py

```
@@ -89,6 +89,15 @@ def get_output(self) -> ModelRunnerOutput:
 
         return self.model_runner_output
 
+    def get_kv_connector_output(self) -> KVConnectorOutput | None:
```

Ruff (0.16.3)

[error] Undefined name `KVConnectorOutput`

(F821)